c2corg_ui/templates/outing/helpers/detailed_outings_attributes.html

```diff
@@ -10,11 +10,11 @@
   </div>
   <span class="detail-text accordion">
 
-    % if outing.get('activities') :
+    % if outing.get('activities'):
       <article class="value activities">
         <span translate class="value-title">activities</span><br>
         % for type in outing.get('activities') :
-        <span x-translate  class="green-text">${type}</span>${'' if loop.last else ', '}
+          <span x-translate  class="route-activity icon-${type}" uib-tooltip="{{mainCtrl.translate('${type}')}}" ></span>
```

Comment:
Do you really need the ``x-translate`` directive here? The ``<span>`` has no content.